consensus/spos/bls/v2/subroundEndRound.go

```diff
@@ -150,6 +150,12 @@ func (sr *subroundEndRound) receivedInvalidSignersInfo(_ context.Context, cnsDta
 		return false
 	}
 
+	invalidSignersHash := sr.Hasher().Compute(string(cnsDta.InvalidSigners))
+	invalidSignersCache := sr.InvalidSignersCache()
+	if invalidSignersCache.HasInvalidSigners(string(invalidSignersHash)) {
```

Comment:
if there are different number of invalidSigners: first time 2: A and B, and in on a second message it gets only A, it seems that the message will be propagated 2 times?